data/social-media.json

```diff
@@ -0,0 +1,18 @@
+{
+    "accounts": [{
```

Comment:
Idea is to concentrate certain information in a single point. Either here as JSON (if other ressources need it, too) or via `.Site.Params`.